### PR TITLE
CTCP-4474: Remove application-ie8 stylings

### PR DESCRIPTION
**Pull request by joecefnhedog**

### CODE REVIEW

**Comment by joecefnhedog** [comment]
> * `uk.gov.hmrc.webjars.hmrc-frontend` in [project/AppDependencies.scala](https://github.com/hmrc/declare-transit-movement-departure-frontend/blob/CTCP-4474/project/AppDependencies.scala#L20) can be removed - its already provided by `uk.gov.hmrc.play-frontend-hmrc`

I don't see `play-frontend-hmrc` and removing `uk.gov.hmrc.webjars.hmrc-frontend` causes errors

File: app/assets/stylesheets/partials/_shame.scss [hunk 1]
```diff
@@ -59,27 +59,6 @@
...
-    width: 40%;
-}
-.ctc-add-to-a-list .govuk-summary-list__key,
-.ctc-add-to-a-list .govuk-summary-list__value {
-    word-wrap: break-word;
-    overflow-wrap: break-word;
-    hyphens: auto;
-}
-
-
```
Comment (jackheslop96) [inline app/assets/stylesheets/partials/_shame.scss:82]: I think this style might be used in this service? Can you double check?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -1,8 +1,7 @@
-$govuk-assets-path: "/manage-transit-movements-departures/assets/lib/govuk-frontend/govuk/assets/";
-$hmrc-assets-path: "/manage-transit-movements-departures/assets/lib/hmrc-frontend/hmrc/assets/";
+$govuk-include-default-font-face: false;
 
 @import "lib/govuk-frontend/govuk/all";
```
Comment (karahanozturk) [inline app/assets/stylesheets/application.scss:3]: do we need to keep lib/govuk-frontend/govuk/all?

Comment (joecefnhedog) [reply app/assets/stylesheets/application.scss:3]: We do for this case, I put a comment on the bottom of the ticket: https://jira.tools.tax.service.gov.uk/browse/CTCP-4474

**Review by karahanozturk — APPROVED** [review]

**Pull request merged**